.github/workflows/ci.yaml

```diff
@@ -0,0 +1,33 @@
+name: CI
+on: push
+
+jobs:
+  rspec:
+    runs-on: ubuntu-latest
+    container: public.ecr.aws/docker/library/ruby:3.2.2
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install Deps
+        run: |
+          apt update && apt install curl -y
```

Comment:
Could either move this `apt update && apt install ffmpeg -y` to the following step, or just replace the `curl -y` with `ffmpeg -y`!